wagtail/contrib/forms/templates/wagtailforms/list_submissions.html

```diff
@@ -1,37 +1,48 @@
-                {% for heading in data_headings %}
-                    <th id="{{ heading.name }}" class="{% if heading.order %}ordered icon {% if heading.order == 'ascending' %}icon-arrow-up-after{% else %}icon-arrow-down-after{% endif %}{% endif %}">
-                        {% if heading.order %}<a href="?order_by={% if heading.order == 'ascending' %}-{% endif %}{{ heading.name }}">{{ heading.label }}</a>{% else %}{{ heading.label }}{% endif %}
-                    </th>
-                {% endfor %}
-            </tr>
-        </thead>
-        <tbody>
-            {% for row in data_rows %}
+{% block results %}
+    <form class="w-overflow-auto" data-controller="w-bulk" data-w-bulk-action-inactive-class="w-invisible" action="{% url 'wagtailforms:delete_submissions' form_page.id %}" method="get">
+        <table class="listing">
+            <col />
+            <col />
+            <col />
+            <thead>
                 <tr>
-                    <td>
-                        <input type="checkbox" name="selected-submissions" class="select-submission" value="{{ row.model_id }}" data-action="w-bulk#toggle" data-w-bulk-target="item" />
-                    </td>
-                    {% for cell in row.fields %}
-                        <td>
-                            {{ cell }}
-                        </td>
+                    <th><input type="checkbox" data-action="w-bulk#toggleAll" data-w-bulk-target="all" /></th>
+                    {% for heading in data_headings %}
+                        <th id="{{ heading.name }}" class="{% if heading.order %}ordered icon {% if heading.order == 'ascending' %}icon-arrow-up-after{% else %}icon-arrow-down-after{% endif %}{% endif %}">
+                            {% if heading.order %}<a href="?order_by={% if heading.order == 'ascending' %}-{% endif %}{{ heading.name }}">{{ heading.label }}</a>{% else %}{{ heading.label }}{% endif %}
+                        </th>
                     {% endfor %}
                 </tr>
-            {% endfor %}
-        </tbody>
-    </table>
-</div>
+            </thead>
+            <tbody>
+                {% for row in data_rows %}
+                    <tr>
+                        <td>
+                            <input type="checkbox" name="selected-submissions" class="select-submission" value="{{ row.model_id }}" data-action="w-bulk#toggle" data-w-bulk-target="item" />
+                        </td>
+                        {% for cell in row.fields %}
+                            <td>
+                                {{ cell }}
+                            </td>
+                        {% endfor %}
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+        <div class="nice-padding">
+            <button class="button no w-invisible" data-w-bulk-target="action">
+                {% trans "Delete selected submissions" %}
+            </button>
+        </div>
```

Comment:
I moved this to the bottom of the table. The old implementation put this as a header row without any visible borders, which I think looks off with the new universal listings.

This should probably use the same "footer" actions like the bulk actions mechanism, but I don't think it's worth spending much time on for now.